### PR TITLE
[MRG] Retry guzzle theme

### PR DESCRIPTION
**Pull request by standage**
See #1630. Retrying guzzle theme for documentation.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?

### CODE REVIEW

**Comment by ctb** [comment]
Failed:

https://readthedocs.org/projects/khmer/builds/5434672/

```
writing output... [ 13%] dev/binary-file-formats

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/cmdline.py", line 296, in main
    app.build(opts.force_all, filenames)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/application.py", line 333, in build
    self.builder.build_update()
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 251, in build_update
    'out of date' % len(to_build))
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 322, in build
    self.write(docnames, list(updated_docnames), method)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 360, in write
    self._write_serial(sorted(docnames), warnings)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/builders/__init__.py", line 368, in _write_serial
    self.write_doc(docname, doctree)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/builders/html.py", line 455, in write_doc
    self.docwriter.write(doctree, destination)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/writers/__init__.py", line 80, in write
    self.translate()
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/writers/html.py", line 48, in translate
    self.document.walkabout(visitor)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 174, in walkabout
    if child.walkabout(visitor):
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 166, in walkabout
    visitor.dispatch_visit(self)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/docutils/nodes.py", line 1882, in dispatch_visit
    return method(node)
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/writers/html.py", line 765, in unknown_visit
    raise NotImplementedError('Unknown node: ' + node.__class__.__name__)
NotImplementedError: Unknown node: todo_node

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/khmer/envs/retry-guzzle/local/lib/python2.7/site-packages/sphinx/writers/html.py", line 765, in unknown_visit
    raise NotImplementedError('Unknown node: ' + node.__class__.__name__)
NotImplementedError: Unknown node: todo_node
The full traceback has been saved in /tmp/sphinx-err-BGLXX8.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!

```

**Comment by standage** [comment]
I tried to recreate the RTD environment on AWS (Ubuntu machine, python2, Sphinx 1.5.3), but I could not reproduce the error. I'm removing a TODO item to confirm whether this is indeed the issue.

File: doc/dev/binary-file-formats.rst [hunk 1]
```diff
@@ -147,5 +147,3 @@ Table size          8         0   Length of table, **in bits** (``uint64_t``).
 Bins                N/8+1     8   This table's bytes, length given by previous
                                   field, divided by 8, plus 1 (``uint8_t``).
 ================== ======= ===== ==============================================
-
-.. todo:: Document ``Tags``, ``Stoptags``, ``Subset``, ``Labelset``
```
Comment (standage) [inline doc/dev/binary-file-formats.rst]: It builds just fine when this line is removed. See https://readthedocs.org/projects/khmer/builds/5435660/.

This definitely seems like a bug/version problem on ReadTheDocs' end, but probably not worth investing any time troubleshooting. I propose we move this TODO to an issue (I assume it's still relevant) and keep this change. Any comments?

**Comment by ctb** [comment]
+1

--
Titus Brown, ctbrown@ucdavis.edu

File: doc/dev/binary-file-formats.rst (hunk 1 above)
Comment (standage) [inline doc/dev/binary-file-formats.rst]: Issue added. #1698

**Comment by standage** [comment]
Read for review and merge! @ctb @betatim

**Comment by ctb** [comment]
ahh.... so nice :) https://khmer.readthedocs.io/en/retry-guzzle/

**Pull request merged**